src/openloco/localisation/string_ids.h

```diff
@@ -421,13 +445,69 @@ namespace openloco::string_ids
     constexpr string_id music_running_on_time = 1622;
     constexpr string_id music_get_me_to_gladstone_bay = 1623;
     constexpr string_id music_sandy_track_blues = 1624;
+    constexpr string_id uint16_raw = 1625;
+    constexpr string_id start_year = 1626;
+    constexpr string_id label_generate_random_landscape_when_game_starts = 1627;
+    constexpr string_id tooltip_generate_random_landscape_when_game_starts = 1628;
+    constexpr string_id button_generate_landscape = 1629;
+    constexpr string_id tooltip_generate_random_landscape = 1630;
+    constexpr string_id label_ok = 1631;
+    constexpr string_id title_generate_new_landscape = 1632;
+    constexpr string_id title_random_landscape_option = 1633;
+    constexpr string_id prompt_confirm_generate_landscape = 1634;
+    constexpr string_id prompt_confirm_random_landscape = 1635;
+    constexpr string_id sea_level = 1636;
+    constexpr string_id sea_level_units = 1637;
+    constexpr string_id number_of_forests = 1638;
+    constexpr string_id number_of_forests_value = 1639;
+    constexpr string_id minimum_forest_radius = 1640;
+    constexpr string_id minimum_forest_radius_blocks = 1641;
+    constexpr string_id maximum_forest_radius = 1642;
+    constexpr string_id maximum_forest_radius_blocks = 1643;
+    constexpr string_id minimum_forest_density = 1644;
+    constexpr string_id minimum_forest_density_pct = 1645;
```

Comment:
What does `pct` mean?

Reply:
Percentage. Good enough, or rename?

Reply:
Rename. Also do expand the `min_` and `max_` prefixes below (for sake of consistency)

Reply:
I think I'd rather contract the `minimum_` and `maximum_` ones instead — they appear to be the minority.

Reply:
Fine with me